{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2013-4859",
      "ASSIGNER": "cve@mitre.org"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-276"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/86196",
          "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/86196",
          "refsource": "MISC",
          "tags": [
            "VDB Entry"
          ]
        },
        {
          "url": "http://www.exploit-db.com/exploits/27284",
          "name": "http://www.exploit-db.com/exploits/27284",
          "refsource": "MISC",
          "tags": [
            "Exploit",
            "Third Party Advisory",
            "VDB Entry"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "INSTEON Hub 2242-222 lacks Web and API authentication"
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "AND",
        "cpe_match": []
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:M/Au:N/C:C/I:C/A:C",
        "accessVector": "NETWORK",
        "accessComplexity": "MEDIUM",
        "authentication": "NONE",
        "confidentialityImpact": "COMPLETE",
        "integrityImpact": "COMPLETE",
        "availabilityImpact": "COMPLETE",
        "baseScore": 9.3
      },
      "severity": "HIGH",
      "exploitabilityScore": 8.6,
      "impactScore": 10,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2019-12-27T17:15Z",
  "lastModifiedDate": "2020-01-09T18:09Z"
}